src/constats/data.js

```diff
@@ -0,0 +1,22 @@
+// @TODO Put some data here
+
+const data = {
+    buttons: [{
+        title:'Certification',
```

Comment:
Remove this
We have to delete this folder
(constats)
so we dont need this file anymore